app/src/androidTest/java/ch/epfl/sdp/DataForDemo.java

```diff
@@ -11,6 +14,7 @@
 import ch.epfl.sdp.contamination.DataSender;
 import ch.epfl.sdp.contamination.GridFirestoreInteractor;
 import ch.epfl.sdp.contamination.Layman;
+import kotlin.NotImplementedError;
```

Comment:
could also use UnsupportedOperationException, standard to java, but that does the job too :-)

Reply:
You're right, will use that one